src/components/reservations/Reservations.js

```diff
@@ -0,0 +1,54 @@
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useNavigate } from 'react-router-dom';
+import { getReservationsThunk } from '../../redux/reservationsSlice';
+import locationIcon from '../../img/location_icon.png';
+import calendarIcon from '../../img/calendar_icon.png';
+import { getCarsThunk } from '../../redux/cars/carsSlice';
+import isUserSigned from '../../helpers/auth';
+
+export default function Reservations() {
```

Comment:
- [ ] No matter how many users I create, I always have 2 reservations without having created any of them:

![image](https://user-images.githubusercontent.com/57421823/223579349-965f937b-9985-4cf7-ae95-29ca43250b0e.png)

Please, ensure that only the reservations the users has created appear on this section. 

- [ ] Also, if I create reservations, they won't appear in "my reservations":

## Context

When I create a reservation your app says that it's ok:

![image](https://user-images.githubusercontent.com/57421823/223579774-ec7842f8-5d9b-4a97-ba3a-ee627b996f64.png)

The server reports 200 ok:

![image](https://user-images.githubusercontent.com/57421823/223579926-46e5f18a-a8d7-43d7-bc9d-8e7960b2ecf1.png)

But "My reservations" reports only the same 2 default reservations, and not my actual created reservations:

![image](https://user-images.githubusercontent.com/57421823/223580169-29d3d369-60c2-42d3-a5f5-d0fe869463a3.png)

If this is a bug, kindly fix it so your app works as intended. 💪